### PR TITLE
[test/plugin] immigrate test oracle scenario.

### PR DESCRIPTION
**Pull request by aderm**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [x] New feature provided
- [ ] Improve performance

- Related issues
#3583 
___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

File: .github/workflows/plugins-test.yaml [hunk 1]
```diff
@@ -494,3 +494,34 @@ jobs:
         run: ./mvnw -f test/plugin/pom.xml clean package -DskipTests docker:build -DBUILD_NO=local >/dev/null
       - name: Run mysql 5.1.2-8.0.15 (53)
         run: bash test/plugin/run.sh mysql-scenario
+
+  Oracle:
```
Comment (wu-sheng) [inline .github/workflows/plugins-test.yaml]: Oracle just have one version to test, so you could add this to a faster group, rather than add a new one.

Comment (aderm) [reply .github/workflows/plugins-test.yaml]: ok

File: .github/workflows/plugins-test.yaml [hunk 2]
```diff
@@ -465,8 +465,16 @@ jobs:
           java-version: 8
       - name: Build SkyWalking Agent
         run: ./mvnw clean package -DskipTests -Pagent >/dev/null
+      - name: Download apm-oracle-10.x-plugin-1.0.1.jar to agent plugins dir.
```
Comment (dmsolr) [inline .github/workflows/plugins-test.yaml]: we can configure multi-command with `run:.`
I suggest,
``` 
- name: Run oracle 10.2.0.4.0 (1)
   run: |
       curl -L -o ...
       curl -L -o ...
       ./mvnw ...
       ./mvnw ...
       bash ....
```

WDYT?

Comment (aderm) [reply .github/workflows/plugins-test.yaml]: ACK

**Pull request merged**